src/main/java/CREResultsRetriever.java

```diff
@@ -63,5 +69,33 @@ private void prepareCREResults() throws IOException {
                 }
             }
         }
+        openFile();
```

Comment:
Hmm, in my opinion will be better to make something like:
File file = openFile(name);
addResultsToFile(file);
closeFile(file);

In that approach methods opneFile and close File can be even in separate class responsible for operations on file.